maven/maven.bzl

```diff
@@ -259,6 +259,8 @@ def _maven_jvm_artifact(artifact_spec, name, visibility, deps = [], use_jetifier
     should_jetify = (use_jetifier and
         coordinate not in JETIFIER_EXCLUDED_ARTIFACTS and
         artifact.group_id != "org.bouncycastle" and
+        artifact.group_id != "com.android.tools" and
```

Comment:
Hmm.  This might work for all of com.android.tools, but let's generalize separately.